### PR TITLE
Add Chao to OWNERS

### PR DESCRIPTION
**Pull request by adrcunha**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -4,6 +4,7 @@ approvers:
 - adrcunha
 - dushyanthsc
 - ericKlawitter
+- chaodaig
```
Comment (srinivashegde86) [inline OWNERS]: can you add nikhil as well :)

**Comment by srinivashegde86** [comment]
/lgtm
/hold
cancel and merge if we want to create another CL to add nikhil.

**Comment by adrcunha** [comment]
/hold cancel

Let's do one thing at a time.

File: OWNERS (hunk 1 above)
Comment (adrcunha) [reply OWNERS]: I prefer doing that later.

**Pull request merged**